client/scripts/com/monsters/maproom_advanced/PopupAttackA.as

```diff
@@ -142,12 +142,12 @@ package com.monsters.maproom_advanced
             GLOBAL._currentCell = this._cell;
             if(this._cell._base == 1)
             {
-               BASE.LoadBase(null,0,this._cell._baseID,"wmattack",false,EnumYardType.MAIN_YARD);
+               BASE.LoadBase(null,0,this._cell._baseID,"wmattack",false,EnumYardType.MAIN_YARD, this._cell._cell_id);
             }
             else
             {
                _loc2_ = this._cell._base == 3 ? int(EnumYardType.OUTPOST) : int(EnumYardType.MAIN_YARD);
-               BASE.LoadBase(null,0,this._cell._baseID,GLOBAL.e_BASE_MODE.ATTACK,false,_loc2_);
+               BASE.LoadBase(null,0,this._cell._baseID,GLOBAL.e_BASE_MODE.ATTACK,false,_loc2_, this._cell._cell_id);
```

Comment:
Reminder: Remove any new client side logic, we should emulate the original server.